composer.json

```diff
@@ -65,8 +65,17 @@
         ],
         "post-update-cmd": [
             "@auto-scripts"
+        ],
+        "setup-clean-db": [
+            "bin/console doctrine:database:drop --if-exists --force",
+            "bin/console doctrine:database:create --if-not-exists",
```

Comment:
--if-not-exists not needed, if the database is force-dropped in the previous call ;)

Reply:
without --if-not-exists I always run in an error message if the table is not exists